ee/vellum_ee/workflows/display/tests/workflow_serialization/test_basic_conditional_node_serialization.py

```diff
@@ -285,7 +285,7 @@ def test_serialize_workflow():
                     {
                         "id": "de7b0b4e-7803-4d36-a275-2e7e3f60342b",
                         "type": "IF",
-                        "source_handle_id": "561b4e3a-8db3-448a-8933-1115937082ff",
```

Comment:
[oos] wow - these have been busted since launch.

piggy backs off of a convo we were having in the elevator. This assertion is stronger if we tested the fields we cared about instead of the full json data